stage1/init/kvm/hypervisor/none_driver.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build hv_none
+
+package hypervisor
+
+import (
+	"errors"
+	"github.com/coreos/rkt/stage1/init/kvm"
+)
+
+func StartCmd(wdPath, name, kernelPath string, nds []kvm.NetDescriber, cpu, mem int64, debug bool) ([]string, error) {
+	return nil, errors.New("You were compiled rkt w/o kvm support and You cannot run containers with kvm flavor")
+}
```

Comment:
@woodbor @jellonek: This function is required in one specific corner case: you are building rkt without kvm support and then you are trying to build `stage1` for kvm manually. In this case our `init` is compiled w/o support and hasn't got compiled-in driver for hypervisor.